cylc/flow/parsec/fileparse.py

```diff
@@ -497,6 +498,9 @@ def read_and_proc(
             flines = jinja2process(
                 fpath, flines, fdir, template_vars
             )
+        elif template_vars.keys() - {"CYLC_VERSION", "CYLC_TEMPLATE_VARS"}:
```

Comment:
Does cylc-rose provide default template vars even if templating was not configured?

Reply:
If you turn Cylc Rose on it provides template variables. You turn it on by adding a blank `rose-suite.conf`.

Reply:
I don't think there's a simple/sane way around that. If you think that's a deal breaker IMO we just call this PR off. It's too small an issue for me to spend much more time on it.